Gems/PhysX/Code/Source/ArticulationLinkComponent.cpp

```diff
@@ -126,6 +126,19 @@ namespace PhysX
             {
                 sceneInterface->RegisterSceneSimulationFinishHandler(m_attachedSceneHandle, m_sceneFinishSimHandler);
 
+                // Create a handler that in the case that the scene was removed before the deactivation of the component,
+                // ensures that all articulations are destroyed.
+                m_sceneRemovedHandler = AzPhysics::SystemEvents::OnSceneRemovedEvent::Handler(
+                    [this]([[maybe_unused]] AzPhysics::SceneHandle sceneHandle)
```

Comment:
sceneHandle appears to be used
```suggestion
                    [this](AzPhysics::SceneHandle sceneHandle)
```